### DISCUSSION TITLE
有没有可能让一个milvus cluster的每个collection写数据到单独的S3 bucket当中去？

### DISCUSSION THREAD

**Discussion by xiaobingxia-at** [milvus-io/milvus#37347] [Q&A and General discussion] [answered]
请教一下各位，现在我建一个milvus cluster，需要指定一个s3 path。比如：s3://milvus_cluser/cluster_1/
然后，后面的所有index file，logs等等都会在这个path下面。
有没有办法可以指定，collection a的数据到s3://milvus_cluster_collection_a/
collection b的数据到s3://milvus_cluster_collection_b/

**Comment by yhmo** [comment, marked as answer]
根据milvus.yaml里的配置，所有的文件都会写入一个统一的路径下面，路径名默认叫files：
https://github.com/milvus-io/milvus/blob/f190e5d802c7235743074805745670d58f44e9b1/configs/milvus.yaml#L130
如果想要写入不同的路径，只能自己去改源码了。
